creator-node/src/services/stateMachineManager/stateReconciliation/StateReconciliationQueue.js

```diff
@@ -23,36 +26,22 @@ const updateReplicaSetJobProcessor = require('./updateReplicaSet.jobProcessor')
  */
 class StateReconciliationQueue {
   async init() {
-    this.queue = this.makeQueue(
+    const queue = this.makeQueue(
       config.get('redisHost'),
       config.get('redisPort')
     )
-    this.registerQueueEventHandlersAndJobProcessor({
-      queue: this.queue,
+    this.registerQueueEventHandlersAndJobProcessors({
+      queue,
       processManualSync: this.processManualSyncJob.bind(this),
       processRecurringSync: this.processRecurringSyncJob.bind(this),
-      processIssueSyncRequests: this.processIssueSyncRequestsJob.bind(this),
+      processEnqueueSyncRequests: this.processEnqueueSyncRequestsJob.bind(this),
       processUpdateReplicaSets: this.processUpdateReplicaSetsJob.bind(this)
```

Comment:
might be misremembering but i thought we'd settled on every job in this queue handling a single action for a single user?

Reply:
yeah so this one was taking all the "potential" syncs and then enqueuing only the ones that are actually needed (if the clock value on the secondary is less than on the primary). I ended up removing this job entirely and just adding the clock check in the findPotentialSyncs job. I think it's much cleaner now

Reply:
sick